src/DependencyInjection/DI/src/ServiceLookup/CallSiteFactory.cs

```diff
@@ -26,7 +24,7 @@ public CallSiteFactory(IEnumerable<ServiceDescriptor> descriptors)
         {
             _stackGuard = new StackGuard();
             _descriptors = descriptors.ToList();
-            Populate(descriptors);
+            Populate(_descriptors);
```

Comment:
It's kind of weird that descriptors is even a parameter since it's already in a field.